src/ui/styles/app.styl

```diff
@@ -99,6 +98,15 @@
   align-items: center;
   justify-content: center;
 
+:global .th *
+  font-family: Noto-Sans-Th, Roboto
+
+:global .ja *
+  font-family: Noto-Sans-Ja
+
+:global .zh *
+  font-family: Noto-Sans-Zh
```

Comment:
It seems that `Noto-Sans-*`, fonts are taller.
Because of this, the dollar amount does not fit in the asset card.